src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/iOS/ProgressBarExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UIKit;
+using Xamarin.Forms;
+
+namespace Xamarin.Platform
+{
+	public static class ProgressBarExtensions
+	{
+		public static void UpdateProgress(this UIProgressView nativeProgressBar, IProgress progress)
+		{
+			nativeProgressBar.Progress = (float)progress.Progress;
+		}
+
+		public static void UpdateProgressColor(this UIProgressView nativeProgressBar, IProgress progress)
+		{
+			nativeProgressBar.ProgressTintColor = progress.ProgressColor == Color.Default ? null : progress.ProgressColor.ToNative();
+		}
+
+		public static void UpdateBackgroundColor(this UIProgressView nativeProgressBar, IProgress progress)
+		{
+			nativeProgressBar.TrackTintColor = progress.BackgroundColor != Color.Default ? progress.BackgroundColor.ToNative() : null;
```

Comment:
NIT: Why is one == and the other !=?

Reply:
Maybe we can get a new extension method for ToNative that takes a param to check? `ToNative(Color? valueIfNull)` so this code can have reduced chance of someone mixing the order of results.